test/extended/util/framework.go

```diff
@@ -1709,10 +1710,11 @@ func (r *podExecutor) Exec(script string) (string, error) {
 	var out string
 	waitErr := wait.PollImmediate(1*time.Second, 3*time.Minute, func() (bool, error) {
 		var err error
-		out, err = r.client.Run("exec").Args(r.podName, "--", "/bin/bash", "-c", script).Output()
+		out, err = e2e.RunHostCmd(r.client.Namespace(), r.podName, script)
```

Comment:
This should have an effect of making the shell output more verbose (`-x`): https://github.com/openshift/origin/blob/f825ad788c95df1b805c89cbdfe912559bf575c2/vendor/k8s.io/kubernetes/test/e2e/framework/util.go#L1108-L1111

I think using `/bin/sh` instead of `/bin/bash` is fine because OpenShift only runs on Red Hat OSes, on which `/bin/sh` is Bash.

Reply:
Agree. Seems like a reasonable change, `/bin/sh` is a link to `bash`.